templatetags/htk_tags.py

```diff
@@ -302,6 +302,28 @@ def get_request_duration():
     return duration
 
 
+@register.simple_tag()
+def localize(key, locale='en-US'):
+
+    from htk.utils import resolve_model_dynamically
+    from django.core.exceptions import ObjectDoesNotExist
+
+    LocalizedString = resolve_model_dynamically(
+        htk_setting('HTK_LOCALIZED_STRING_MODEL')
+    )
+
+    try:
+        localized_string = LocalizedString.objects.get(
+            localizable_string__key=key,
+            language_code=locale,
+        ).value
+
+    except ObjectDoesNotExist:
```

Comment:
```suggestion
    except LocalizedString.DoesNotExist:
```

The `ObjectDoesNotExist` exception is also available as a convenience accessor on the model.